setup.py

```diff
@@ -40,7 +40,7 @@
         "async-generator==1.10",
         "bloom-filter==1.3",
         "cachetools>=2.1.0,<3.0.0",
-        "coincurve>=8.0.0,<9.0.0",
+        "coincurve>=10.0.0,<11.0.0",
```

Comment:
@javimaravillas what's the reason for bumping these?

Reply:
Ah, ok I just read it was suggested [here](https://github.com/ethereum/py-evm/issues/1484) as lower versions have issues with FreeBSD. Sounds fair.